src/org/labkey/test/components/ui/files/FileAttachmentEntry.java

```diff
@@ -0,0 +1,116 @@
+package org.labkey.test.components.ui.files;
+
```

Comment:
A lot of pages have their own implementations of interactions with this, it's possibly a thing we could use to consolidate those implementations into less code  (or, if we don't want to, not)